k8s/production/karpenter/provisioners/gitlab/provisioner.yaml

```diff
@@ -12,15 +12,10 @@ spec:
         spack.io/node-pool: gitlab
     spec:
       requirements:
-        - key: "node.kubernetes.io/instance-type"
+        # Always use amd64 nodes
```

Comment:
Is there a particular reason for this? I'm guessing the gitlab helm chart just doesn't run any arm images?

Reply:
Yes, you are correct. Relevant upstream issues:

https://gitlab.com/gitlab-org/omnibus-gitlab/-/issues/5673
https://gitlab.com/groups/gitlab-org/-/epics/2370